configure.ac

```diff
@@ -653,12 +654,14 @@ case "${host}" in
 	SOREV="${rev}.${so}"
 	sbrk_deprecated="1"
 	SYM_PREFIX="_"
+	zero_realloc_default_alloc="1"
```

Comment:
I don't know that this is right, but I'll trust that you checked it.

Reply:
Yeah the [Mac OS man page](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/realloc.3.html) is clear: `If size is zero and ptr is not NULL, a new,
     minimum sized object is allocated and the original object is freed.`